research/delf/delf/python/datasets/sfm120k/dataset_download.py

```diff
@@ -15,11 +15,40 @@
 """Structure-from-Motion dataset (Sfm120k) download function."""
 
 import os
-import wget
+import urllib3
 import tarfile
+import requests
+from tqdm import tqdm
 
 import tensorflow as tf
 
+# Suppress SSL Warnings
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+
+def wget(src, dst, md5checksum=False):
+    """Download a file and calculate it's MD5 chacksum"""
+
+    # Writing in chunks helps in preventing running out of Memory
+    with requests.get(src, stream=True, verify=False) as r:
+        r.raise_for_status()
+        with open(dst, 'wb') as f, tqdm(desc="Downloading",
+        unit='iB',
+        unit_scale=True,
+        unit_divisor=1024,
+    ) as bar:
+            for chunk in r.iter_content(chunk_size=8192):
+                size = f.write(chunk)
+                bar.update(size)
+
+    if md5checksum:
```

Comment:
Hmm, MD5 is broken, can we switch to SHA instead?

Reply:
Done! Replaced MD5 with SHA256 😃